geobuf/decode.py

```diff
@@ -136,34 +139,42 @@ def decode_line(self, coords, is_closed=False):
         return obj
 
     def decode_multi_line(self, geometry, is_closed=False):
-        if len(geometry.lengths) == 0:
-            return [self.decode_line(geometry.coords, is_closed=is_closed)]
-
         obj = []
-        i = 0
 
-        for l in geometry.lengths:
-            obj.append(self.decode_line(geometry.coords[i:i + l * self.dim], is_closed=is_closed))
-            i += l * self.dim
+        if len(geometry.coords) == 0:
+            return obj
```

Comment:
Could keep the single return pattern by joining this `if` with the block below and `pass`ing